pages/linux/rig.md

```diff
@@ -0,0 +1,24 @@
+# rig
+
+> Utility that will piece together a random first name, last name, street number and address, along with a geographically consistant (ie, they all match the same area) city, state, ZIP code, and area code.
+> More information: <http://manpages.ubuntu.com/manpages/bionic/man6/rig.6.html>.
+
+- Return a random name and address:
```

Comment:
Maybe we can say here the randomly gender to get a simpler example description is the next example:
```suggestion
- Display a random name (male or female) and address:
```